src/integrations/prefect-dbt/prefect_dbt/core/runner.py

```diff
@@ -114,6 +112,9 @@ class PrefectDbtRunner:
             non-exception failure
         client: Optional Prefect client instance
         include_compiled_code: Whether to include compiled code in the asset description
+        enable_assets: Global override for disabling asset generation for dbt nodes. If
```

Comment:
i think we need to change this language or make the flag `disable_assets=False`

```suggestion
        enable_assets: Global override for enabling asset generation for dbt nodes. If
```

Reply:
lol I had it as `disable_assets` at first and then changed it.